app/src/main/java/com/jetbrains/handson/mpp/mobile/MainActivity.kt

```diff
@@ -92,21 +97,47 @@ class MainActivity : AppCompatActivity(), ApplicationContract.View {
     }
 
     fun buttonClick(view: View) {
-        if (autotextDep.text.toString() == "") {
+        val depText = autotextDep.text.toString()
```

Comment:
really minor thing, doesn't even really warrant a change, but personally I try and avoid variable names like this as much as possible. "departureText" and "arrivalText" hardly take up more space and mean the variable can just be read as it is rather than needing contextual knowledge, If that makes sense?

I think the best way I've heard it argued is "It's not like we have a budget on characters"

Reply:
Good point - I agree. I'll change it now as it'll be a quick one to do.